admin/helpers/logger.js

```diff
@@ -12,7 +12,7 @@ const init = (app) => {
  * Logging
  * use LogDNA transport for winston if configuration setting available
  */
-  if (config.Logging.LogDna.key) {
+/*   if (config.Logging.LogDna.key) {
```

Comment:
Why not remove it.

Reply:
it should have been.  that was temporary while i checked for other references.

removing...